rebench/tests/persistency_test.py

```diff
@@ -168,3 +170,44 @@ def _exec_rebench_db(self, cmd_config, server):
 
         run = list(cnf.get_runs())[0]
         run.close_files()
+
+    def test_check_file_lines(self):
+        ds = DataStore(self.ui)
+        cnf = Configurator(load_config(self._path + '/persistency.conf'),
+                            ds, self.ui, data_file=self._tmp_file)
+        ds.load_data(None, False)
+        ex = Executor(cnf.get_runs(), False, self.ui)
+        ex.execute()
+        with open(self._tmp_file, 'r') as file: # pylint: disable=unspecified-encoding
+            lines = file.readlines()
+            command = self.get_line_after_char('#!', lines[0])
+            self.assertEqual(command, subprocess.list2cmdline(sys.argv))
+            time = self.get_line_after_char('Start:', lines[1])
+            self.assertTrue(self.is_valid_time(time))
+            json_code = self.get_line_after_char('Environment:', lines[2])
+            self.assertTrue(self.is_valid_json(json_code))
+            json_code = self.get_line_after_char('Source:', lines[3])
+            self.assertTrue(self.is_valid_json(json_code))
+            line = lines[4].strip().split()
+            words = Measurement.get_column_headers()
+            self.assertEqual(line, words)
```

Comment:
Ok, so, does the next line have the same number of columns as the header? :) 
Also, two lines up, you use `split()` without actually giving it a separator. Does this work as expected?